schema.graphql

```diff
@@ -501,13 +521,15 @@ type Mutation {
   removeDirectChat(chatId: ID!, organizationId: ID!): DirectChat!
   removeEvent(id: ID!): Event!
   removeEventAttendee(data: EventAttendeeInput!): User!
+  removeEventProject(id: ID!): EventProject!
   removeGroupChat(chatId: ID!): GroupChat!
   removeMember(data: UserAndOrganizationInput!): Organization!
   removeOrganization(id: ID!): User!
   removeOrganizationCustomField(customFieldId: ID!, organizationId: ID!): OrganizationCustomField!
   removeOrganizationImage(organizationId: String!): Organization!
   removePost(id: ID!): Post
   removeSampleOrganization: Boolean!
+  removeTask(id: ID!): Task
```

Comment:
@duplixx 

1. We removed Projects and Tasks from the API a few PRs ago.  Your starting code is out of date
    - https://github.com/PalisadoesFoundation/talawa-api/pull/1627
1. This is going to be replaced with Action Items.
    - https://github.com/PalisadoesFoundation/talawa-api/pull/1648
1. Remove this Type and all references to it